CreateSnapshot/build.gradle

```diff
@@ -10,6 +10,7 @@ java.targetCompatibility = JavaVersion.VERSION_11
 
 dependencies {
```

Comment:
Can we enforce this via the subprojects definition rather than on each project seperately, seems like we are building up alot of copypasta in our individual projects?

Reply:
Addressed, thanks